docs/appendix/glossary/model-namespace-triplet.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Model Namespace Triplet
+id: model-namespace-triplet
+full_link:
+short_description: namespace:family:name
+aka:
+---
+
+{{< glossary_tooltip term_id="model" text="Models" >}} are uniquely namespaced as colon-delimited-triplets in the form of `namespace:family:name`.
+
+For example:
+
+- The `rdk:builtin:gpio` model of the `rdk:component:motor` API provides RDK support for [GPIO-controlled DC motors](/components/motor/gpio/).
+- The `rdk:builtin:DMC4000` model of the same `rdk:component:motor` API provides RDK support for the [DMC4000](/components/motor/dmc4000/) motor.
```

Comment:
we probably want to take out RDK here and implementation may be a better word than support here

Reply:
Ooh this is a tricky RDK usage. Since "rdk" is part of the triplet, maybe it actually makes sense to leave it in here for context? Also I think "support" makes sense to me here because I think of it as, the code is the implementation that supports the hardware. In other words, `rdk:builtin:gpio` is an implementation of the `rdk:component:motor` API, and that implementation of that API supports a given motor.

Here's a suggestion if "RDK" comes out:
```suggestion
- The `rdk:builtin:gpio` model of the `rdk:component:motor` API supports [GPIO-controlled DC motors](/components/motor/gpio/).
- The `rdk:builtin:DMC4000` model of the same `rdk:component:motor` supports the [DMC4000](/components/motor/dmc4000/) motor controller.
```
Maybe the following would use the word "implement" correctly?
```suggestion
- The `rdk:builtin:gpio` model implements the `rdk:component:motor` API to support [GPIO-controlled DC motors](/components/motor/gpio/).
- The `rdk:builtin:DMC4000` model of the same `rdk:component:motor` supports the [DMC4000](/components/motor/dmc4000/) motor controller.
```

Reply:
I think this suggestion looks good but @mcvella as author does this change make sense to you?

Reply:
how about saying "provides built-in" support?  This way we don't have to say RDK but imply that it is part of RDK/viam-server?  Otherwise its fine as-is, since some of these things might become modular in the future anyway!

Reply:
```suggestion
- The `rdk:builtin:gpio` model implements the `rdk:component:motor` API for [GPIO-controlled DC motors](/components/motor/gpio/).
- The `rdk:builtin:DMC4000` model implements the `rdk:component:motor` API for [DMC4000](/components/motor/dmc4000/) motor controllers.
```